testsupport/init.go

```diff
@@ -109,7 +111,19 @@ func WaitForDeployments(t *testing.T) wait.Awaitilities {
 		// Wait for the webhooks in Member 1 only because we do not deploy webhooks for Member 2
 		// (we can't deploy the same webhook multiple times on the same cluster)
 		// Also verify the autoscaling buffer in both members
-		initMemberAwait.WaitForMemberWebhooks()
+
+		var webhookImage string
+		for _, container := range memberDeployment.Spec.Template.Spec.Containers {
+			if container.Name == "manager" {
+				for _, env := range container.Env {
+					if env.Name == "MEMBER_OPERATOR_WEBHOOK_IMAGE" {
+						webhookImage = env.Value
```

Comment:
you could probably use a `break` to exit the `for` loops (but it's a minor improvement)

Reply:
good point a7b318b